src/specs/components/site-header/demo/data--eu.js

```diff
@@ -29,6 +29,16 @@ module.exports = {
     code: 'en',
     overlay: {
       title: 'Select your language',
+      close: {
+        variant: 'ghost',
+        label: 'Close',
+        icon: {
+          path: '/icons.svg',
+          name: 'close-filled',
```

Comment:
tell me that you protested about this, (the different icon in EC and EU) this shouldn't happen, it has a minimal business value, ok, but an high risk of mistakes by the users.

Reply:
I raised it when working on the notification, but I guess that it is worth raising it again